app/backend/src/couchers/phone/sms.py

```diff
@@ -17,4 +20,33 @@ def send_sms(number, message):
     """Send SMS to a E.164 formatted phone number. Return "success" on
     success, "unsupported operator" on unsupported operator, and any other
     string for any other error."""
-    return "unsupported operator"
+
+    if not config["ENABLE_SMS"]:
+        logger.info(f"SMS not emabled, need to send to {number}: {message}")
+        return
+
+    if len(request.message) > 140:
+        return "message too long"
```

Comment:
```suggestion
    assert len(request.message) <= 140
```

Reply:
I think it's good to use asserts for asserting things that should be true. E.g. making sure an invariant is right, but it's not good to use asserts to verify business logic. Asserts can be turned off with a command line switch "for performance", etc.

Reply:
But it is a programming error if we generate a too long message. Whatever user input should not cause the sms to become too long.